code/game/objects/items/crayons.dm

```diff
@@ -24,55 +24,83 @@
 
-	var/charges = 30 //-1 or less for unlimited uses
+	/// Number of times this item can be used, -1 for unlimited
+	var/charges = 30
+	/// Number of remaining charges
 	var/charges_left
-	var/volume_multiplier = 1 // Increases reagent effect
-
+	/// Multiplies effect of reagent when applied to mobs or surfaces
+	var/volume_multiplier = 1
+	/// If true, sprayed turfs should also have the internal chemical applied to them
+	var/expose_turfs = FALSE
+	/// If set to false, this just applies a chemical and cannot paint symbols
 	var/actually_paints = TRUE
 
+	/// If false a do_after is required to draw something, otherwise it applies immediately
 	var/instant = FALSE
-	var/self_contained = TRUE // If it deletes itself when it is empty
+	/// If true, this deletes itself when empty
+	var/self_contained = TRUE
 
-	var/edible = TRUE // That doesn't mean eating it is a good idea
+	/// Whether or not you can eat this. Doesn't mean it is a good idea to eat it.
+	var/edible = TRUE
 
+	/// Reagents which are applied to things you use this on, or yourself if you eat it
 	var/list/reagent_contents = list(/datum/reagent/consumable/nutriment = 0.5)
-	// If the user can toggle the colour, a la vanilla spraycan
+	/// If the user can toggle the colour, a la vanilla spraycan
 	var/can_change_colour = FALSE
 
+	/// Whether this item has a cap that can be toggled on and off
 	var/has_cap = FALSE
+	/// Whether the cap is currently on or off
 	var/is_capped = FALSE
 
+	/// Whether to play a sound before using
 	var/pre_noise = FALSE
+	/// Whether to play a sound after using
 	var/post_noise = FALSE
 
+	/// List of selectable graffiti options
+	var/static/list/graffiti = list("amyjon","face","matt","revolution","engie","guy","end","dwarf","uboa","body","cyka","star","poseur tag","prolizard","antilizard")
+	/// List of selectable symbol options
+	var/static/list/symbols = list("danger","firedanger","electricdanger","biohazard","radiation","safe","evac","space","med","trade","shop","food","peace","like","skull","nay","heart","credit")
+	/// List of selectable drawing options
+	var/static/list/drawings = list("smallbrush","brush","largebrush","splatter","snake","stickman","carp","ghost","clown","taser","disk","fireaxe","toolbox","corgi","cat","toilet","blueprint","beepsky","scroll","bottle","shotgun")
+	/// List of selectable orientable options
+	var/static/list/oriented = list("arrow","line","thinline","shortline","body","chevron","footprint","clawprint","pawprint") // These turn to face the same way as the drawer
+	/// List of selectable rune options
+	var/static/list/runes = list("rune1","rune2","rune3","rune4","rune5","rune6")
+	/// List of selectable random options
+	var/static/list/randoms = list(RANDOM_ANY, RANDOM_RUNE, RANDOM_ORIENTED,
+		RANDOM_NUMBER, RANDOM_GRAFFITI, RANDOM_LETTER, RANDOM_SYMBOL, RANDOM_PUNCTUATION, RANDOM_DRAWING)
+	/// List of selectable large options
+	var/static/list/graffiti_large_h = list("yiffhell", "furrypride", "secborg", "paint")
```

Comment:
I have done this with the one exception of 
```
"danger",
"electricdanger",
"firedanger",
"evac",
```
I left this slightly out of order because this is the order it displays in the tab and it makes sense for three variants of the same thing to be next to each other, but if you really want I guess I could also rename the icon state so they can stay in alphabetical order and also be next to each other.

Reply:
that's okay then but might want to have a comment saying why it's like that so someone in the future doesn't go "haha old coder screwed something up time to fix it"